### ISSUE TITLE
IIP 2: Improved Personal Performance Report Features

### ISSUE THREAD

**Issue by jesusthatsgreat** [ICNHodler/IIPs#2]
All discussion related to https://github.com/ICNHodler/IIPs/blob/master/IIPS/iip-2.md

**Comment by jesusthatsgreat** [comment]
@RayQuin I agree it would be nice to see all of those proposed stats. In particular, I feel this is priority:

> On the 'Compare' tab on the ICONOMI platform, it is possible to compare any two DAAs with each other. It is not however possible to compare a DAA with an individual Digital Asset, such as Bitcoin or Ethereum.

Most DAA investors do not invest with USD, they invest with BTC or ETH and so presumably they'd at the very least be curious as to their profit/loss in BTC/ETH. At worst they manually work out profit / loss either on paper or with rough calculations in their head (which is what I do, I invested in ETH, therefore I measure profit/loss in ETH first and foremost).

But I really don't think these are just 'nice to have' stats, I think they're also critical for marketing purposes. If a DAA is consistently outperforming BTC/ETH over time, that should be shouted from the rooftops and used to explain the value / importance of diversification. If a DAA is consistently under-performing the market, then both potential investors and the DAA manager should also be aware of that. Several DAA managers specifically say that their goal is to outperform the market - but right now we have no easy way of measuring performance and seeing whether they're achieving that goal or not...

**Comment by RayQuin** [comment]
Yes I do agree with the vs BTC/ETH being a particularly valuable stat. I think in a subsequent iteration of this IIP I would also suggest the option for users to have Euros be the base fiat currency on the platform, especially with ICONOMI's European heritage and fiat gateway, although again this is adding in further fragmentation to the IIP. I may divide it up into separate IIP's, but for now the essentials are there.

Edit: Also I should just say that the GIPS standards under the research heading is just a quick google search result, and may not be at all relevant for DAAs, or practicable for ICONOMI to use - they seem so, but we should probably remove them unless I give them a quick check to see if they are relevant.

**Comment by mvidmar** [comment]
This is a big IIP and contains multiple things. Perhaps it would be better to split into three different IIPs since they would be implemented and released separately in any case and it would make tracking progress easier:
1. Historic user portfolio tracking: total, aggregate digital assets and DAAs
2. Adding BTC, ETH and EUR as a tracking mechanism for current and historic portfolio value
3. DAA comparisons against ETH and BTC 

If we go according the the above I can confirm that features under 1 are on the roadmap for this year. So we can change that status to in progress. 

2 and 3 are a bit more complex. One from an implementation standpoint, the other from an UX perspective - not many users currently use (or find) Compare as a feature when selecting DAAs, so solving the discovery of DAAs is of higher priority. We can confirm however that both are interesting to add in the future so we would categorize them as under consideration.

**Comment by RayQuin** [comment]
Thank you for the kind comment Miha. I will write them up into three separate pieces as you have suggested and ask jesusthatsgreat to implement them with your explanation of the difficulties, so people know what to expect of them.

Thanks,
Ray
